Wordle.Api/Wordle.Api/Models/Score.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Wordle.Api.Models;
+
+[Table("Scores")]
+public class Score
```

Comment:
I can see why a separate table for Scores (one row for each score) might be added, but this looks like a copy of the other one

Reply:
I agree, you are recreating the same table twice, giving a chance for one table to not match the other even though they have the same data

Reply:
I was just messing around and knew this was not the best solution. But thank you for your suggestions.